uuid.gd

```diff
@@ -1,22 +1,26 @@
 # Note: The code might not be as pretty it could be, since it's written
 # in a way that maximizes performance. Methods are inlined and loops are avoided.
-extends Node
+extends Object
```

Comment:
I use a `Node` because if people want to add it as a Singleton to their project, it is the simplest node to use. Is it possible to add an `Object` as a singleton in Godot v4?

Reply:
That is a good point. I didn't think about using `UUID` as singleton. While I think that is not a good way of doing it (why clutter autoload list?), it is still a valid way.

And no, that wasn't changed in 4.0. 

I'll revert this as well.